src/player/Player.php

```diff
@@ -515,6 +519,17 @@ public function hasAutoJump() : bool{
 		return $this->autoJump;
 	}
 
+	public function setFlightSpeed(float $flightSpeed) : void{
+		if($flightSpeed >= 0 && $this->flightSpeed !== $flightSpeed){
```

Comment:
Why should we disallow setting negative flight speed values? Negative values make the player fly backwards, and this can be used in some game modes

Reply:
Yeah you're right